### PR TITLE
Portfolio: Local Storage

### PR DESCRIPTION
**Pull request by OybekKayumov**
Local Storage

Add change event listener on form input
Save form input data on local storage upon change event
Populate form input from local storage on page load
Fix linter errors

### CODE REVIEW

File: scripts/script.js [hunk 1]
```diff
@@ -286,3 +286,35 @@ form.addEventListener('submit', (event) => {
...
+});
+
+window.addEventListener('load', () => {
+  const inputElements = document.querySelectorAll('.contact-form-input');
+  if (localStorage.getItem('formData') !== '' || localStorage.getItem('formData') !== null) {
+    inputElements.forEach((input) => {
+      input.value = JSON.parse(localStorage.getItem('formData'))[input.name];
+    });
+  }
+});
```
Comment (andersonlebon) [inline scripts/script.js:320]: - [ ] GOOD JOB :clap: :100: . I can see the contact form information is persistent to page reloads but only one is left. When the page is reloaded the input field(textarea) containing the message is not populated with the information from the `localstorage`. Kindly let's go ahead and fix it.  check here :point_down:  down what I can see when the page is loaded or reloaded
Screenshot from 2021-12-17 11-34-27

**Review by andersonlebon — CHANGES_REQUESTED** [review]
### Hi @OybekKayumov and @gtekle 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: scripts/script.js (hunk 1 above)
Comment (OybekKayumov) [reply scripts/script.js:320]: Hello,    
Please find our demo here. It works correctly in our side and could you please take a look again. 
https://www.loom.com/share/d583db0be672434fa180cfbb68f121e6

local-storage-snapshot

Thank you.

**Review by julie-ify — APPROVED** [review]
## Hi Team, 

## STATUS: APPROVED ✔️ 

Your project is complete!  all the input fields are persistent on the form after I entered my details and the `textarea` input also persists when it is not on focus. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved emoji


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**